pkgs/by-name/ya/yazi/flavors/gruvbox-dark/default.nix

```
@@ -0,0 +1,24 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+stdenv.mkDerivation rec {
+  name = "gruvbox-dark";
```

I don't think we want to use `name` anymore. Instead, please use a combination of `pname` and `version`.


```suggestion
  pname = "gruvbox-dark";
  version = "0-unstable-2025-12-10";
```